From 0301cb4ffbf4705316ba017516745a535d66d552 Mon Sep 17 00:00:00 2001
From: Bernd Kuhls <bernd.kuhls@t-online.de>
Date: Sun, 28 May 2017 16:56:39 +0200
Subject: [PATCH 1/1] Fix uclibc build
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Without sys/types.h the build breaks with errors like this:

parse_file.h:28:5: error: unknown type name ‘off_t’
     off_t *map;
     ^

Patch sent upstream: https://github.com/fhunleth/fwup/pull/61

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
---
 src/sparse_file.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/sparse_file.h b/src/sparse_file.h
index 21bf5f9..579fbaa 100644
--- a/src/sparse_file.h
+++ b/src/sparse_file.h
@@ -18,6 +18,7 @@
 #define SPARSE_FILE_H
 
 #include <confuse.h>
+#include <sys/types.h>
 
 struct sparse_file_map
 {
-- 
2.11.0

